hello/firmware/cloudToDevice/led.json

```diff
@@ -0,0 +1 @@
+[[1, 1688484111, true]]
```

Comment:
**question**: @maxd-nordic why are the JSON messages always wrapped in an array?

Reply:
All the messages are "from-device-message" or "to-device-message". They are always an array, even if only containing one element.

Reply:
this is to reduce encoding and decoding complexity, and the usual case will be multiple elements

Reply:
The "usual case"? Can you provide an example? Why would the device send multiple temperature readings, button presses, or battery percentages? At any given time there is just one of these.

Reply:
The usual case as I picture it is the following: Due to an event or timeout, the device has sampled data from its sensors and sends it to cloud. Since it's more than one sensor, this will be a list of data points.
Pressing the button is a special case where probably only one element is present (although we might even have more data to send at that moment).

Reply:
Ah, I see. So this is a deviation from asset_tracker_v2's behaviour where it sends individual messages via MQTT.